client/reader/stream/reader-list-followed-sites/index.jsx

```diff
@@ -102,7 +102,19 @@ export class ReaderListFollowedSites extends Component {
 					</h2>
 				) : null }
 				{ sites.length >= searchThreshold && (
-					<FollowingManageSearchFollowed onSearch={ this.searchEvent } initialValue={ query } />
+					<SearchCard
+						compact
+						pinned={ false }
+						className="following-manage__search-followed"
+						additionalClasses="following-manage__search-followed-input"
```

Comment:
Seems you removed the `style.scss` where the `.following-manage__search-followed` were defined.

Reply:
Good find! Brought back the necessary styles in b77f089f65790bcffb6d786e902b1ba3af8651df in the stylesheet that's imported in this particular component.

Note that in the removed styles there were more styles with that classname, but they won't be utilized anymore because they were namespace to a classname that is no longer used.

Can you take another look when you get a chance?